python/lsst/ts/externalscripts/auxtel/latiss_cwfs_align.py

```diff
@@ -462,35 +477,67 @@ async def run_cwfs(self):
         results_dict = self.calculate_results()
         return results_dict
 
+    def get_donut_region(self, center):
```

Comment:
Consider making this receive `center_x` and `center_y` so you don't have to create the local variables... 

When calling with `get_intra_donut_center` you can do

```
self.get_donut_region(*self.get_intra_donut_center())
```

same for extra